File: widgets/bridge_plugin/android/src/main/kotlin/com/facephi/bridge_plugin/sdkmanager/data/request/OnboardingRequest.kt
package com.facephi.bridge_plugin.sdkmanager.data.request

data class OnboardingRequest(
    var operationId: String = "",
    var customerId: String = "",
    var selphiConfigurationData: SelphiConfigurationData,
    var selphIdConfigurationData: SelphIdConfigurationData
)

File: Sources/facephiSDK-ios-package/Request/OnboardingRequest.swift
//
//  File.swift
//
//
//  Created by Faustino Flores García on 19/1/22.
//

import Foundation

public class OnboardingRequest: Codable {
    var operationId: String = ""
    var customerId: String = ""
    var selphiConfigurationData = SelphiConfiguration()
    var selphIdConfigurationData = SelphIdConfiguration()

    public init(operationId: String, customerId: String, selphiConfigurationData: SelphiConfiguration, selphIdConfigurationData: SelphIdConfiguration) {
        self.operationId = operationId
        self.customerId = customerId
        self.selphiConfigurationData = selphiConfigurationData
        self.selphIdConfigurationData = selphIdConfigurationData
    }
}
